code/datums/objective/assasinate_marshal.dm

```diff
@@ -1,30 +1,35 @@
 /datum/objective/assassinate/marshal
 
+
 /datum/objective/assassinate/marshal/get_panel_entry()
 	var/target = src.target ? "[src.target.current.real_name], the [src.target.assigned_role]" : "no_target"
-	return "Find and assassinate <a href='?src=\ref[src];switch_target=1'>[target]</a>."
+	return "You are after the fugitive <a href='?src=\ref[src];switch_target=1'>[target]</a>."
```

Comment:
`panel_entry` shows up in antag panel (only visible for admins). "Find and assassinate" seems like more obvious description.

Reply:
Asassinate is too...evil. 
The marshal is supposed to be one of the good guys, or at least neutral. I'd like to make capturing and jailing the target alive count too